.github/workflows/ci-pipeline.yml

```diff
@@ -44,11 +44,11 @@ jobs:
       run: |
         # install reduce from git
         git clone https://github.com/rlabduke/reduce
```

Comment:
reduce should be clone in ~/src/ folder, otherwise `cmake ~/src/reduce` is not going to work I guess